app/router/student.py

```diff
@@ -125,3 +130,79 @@ async def verify_student(request: Request, student_id: str):
             return HTTPException(status_code=404, detail="Student ID does not exist!")
         return True
     return HTTPException(status_code=response.status_code, detail=response.errors)
+
+
+def get_jnv_code(data):
+    response = requests.get(
+        school_db_url, params={"region": data.region, "name": data.school_name}
+    )
+    if response.status_code == 200:
+        if len(response.json()) == 0:
```

Comment:
in what case will response status_code be 200, but `len(response.json() == 0` be true?

Reply:
When there is no matching entry found in DB.